src/main/java/io/temporal/samples/helloworld/HelloWorldWorker.java

```diff
@@ -0,0 +1,32 @@
+package io.temporal.samples.helloworld;
+// @@@SNIPSTART java-hello-world-sample-worker
+import io.temporal.client.WorkflowClient;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import io.temporal.worker.Worker;
+import io.temporal.worker.WorkerFactory;
+
+public class HelloWorldWorker {
+
+  public static void main(String args[]) {
+    // Create a gRPC stubs wrapper that talks to the local docker instance of the Temporal service
+    WorkflowServiceStubs service = WorkflowServiceStubs.newInstance();
+    // Create a Temporal Java SDK client
+    WorkflowClient client = WorkflowClient.newInstance(service);
+    // Create a Worker factory that can be used to create Workers for specific Task Queues
+    WorkerFactory factory = WorkerFactory.newInstance(client);
+    // Define the name of the Task Queue that the Worker will listen to
+    final String TASK_QUEUE = "java-hello-world-task-queue";
```

Comment:
Instead of duplicating the constant, wouldn't it be better to put as part of a shared type?

Reply:
Not sure if using externally defined variables is a good idea in a tutorial